.github/workflows/build.yml

```diff
@@ -77,7 +77,7 @@ jobs:
       run: exit $(< make-check.status)
 
   experimental:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
The other `runs-on` statement is also for `ubuntu-latest`, but where that name actually comes from?  Wouldn't it make sense to use "fixed" distro version name here too?

Reply:
`ubuntu-latest` is a tag on the Ubuntu Docker image: https://hub.docker.com/_/ubuntu

Apparently it refers to the latest LTS version, i.e. currently 22.04. This is a rather conservative choice, IMHO.

I think this is preferable to pinning to a specific version, like `ubuntu-20.04`, because these tend to get outdated quickly and we need to spend more time than necessary keeping the CI system alive.